cmds/replay.c

```diff
@@ -703,7 +703,10 @@ void get_argspec_string(struct uftrace_task_reader *task,
 
 			if (sym) {
 				print_args(&args, &len, "%s", color_symbol);
-				print_args(&args, &len, "&%s", sym->name);
+				if (format_mode == FORMAT_HTML)
+					print_args(&args, &len, "&amp%s", sym->name);
```

Comment:
Isn't it `&amp;`?

Reply:
The output looked fine, but changed it to `&amp;` anyway.